site/_posts/2020-09-10-building-profile-portal-with-flask-oauth-apis.md

```diff
@@ -819,6 +852,6 @@ If you want to take this tutorial further, here are some improvements:
 
 The code in this application uses both standard OAuth and OIDC to retrieve data. It also uses the FusionAuth APIs. The flask application you built can retrieve and modify users' data, allowing them to make changes to the data provided at registration.
 
-We've reached the end of this blog series about registration forms. You have all the pieces to put together a registration flow for custom applications. You can create the registration form, theme it and provide your user a way to update their profile data.
+We've reached the end of this blog series about registration forms. You now have all the pieces to put together a registration flow for custom applications. You can create the registration form, theme it, and provide your user a way to update their profile data.
```

Comment:
This was written before we had the user profile management feature. Let's add a note to the docs. Something like:

```
If you'd rather have FusionAuth manage the user profile editing page, you can use the [self service user account management feature](/docs/v1/tech/account-management/updating-user-data/). This feature is available in all  paid editions.
``